README.md

```diff
@@ -73,6 +73,23 @@ CheckBuilder now only expects two parameters:
     CheckBuilder[HttpStatusCheckType, Response]
 ```
 
+### New features introduced in Gatling 3.9.0
+Gatling Expression Language (EL) has introduced new random generator functions:
+- randomUUID()
+- randomSecureUuid()
+- randomAlphanumeric()
+- randomInt()
+- randomLong()
+- randomDouble()
+
+These are provided out of the box, you can use them like this in a Gatling expression:
+```
+#{randomInt()}
+```
+
+FeederBuilderBase<T> now has access to a `recordsCount` method, so you do not need to do any extra work in Scala to
+get the length of the records.
+
```

Comment:
oh nice, wonder if we should propose an update to getting started doc - which mentions creating a custom feeder for uuids - maybe we could add a note there like 
> before you create a custom feeder check if gatling already provides a suitable random generator function for your use case. Gatling 3.9.0 introduced randomUUID, randomSecureUuid, ... <<note: are these both right because one is all uppercase one just capitalised>>

Reply:
Good shout, I'll throw in something there too 😁

Reply:
Also with the note asking if they're both right, these come directly from Gatling: https://docs.gatling.io/release-notes/whats-new/3.9/